docs/src/suitable_commands.md

```diff
@@ -0,0 +1,55 @@
+# Suitable commands
+
+### help
+
+### index
+
+- [x] Manage(list, add, delete) custom plugin indexes
```

Comment:
My take is that cli creators will hardcode their indexes. We can add that feature later.

We have two personas:

CLI creator (like ODO - who can use our SDK to bring some extensions they want to use)
CLI user (Person who uses ODO CLI)

My take is that indexes would be controlled by CLI creator and included at build time.

Reply:
You seem right, we can do this later. As other tasks look more important. As of now, we can have only one index and CLI developers can submit their plugins to that index.
later we can support their own hosted indexes

Reply:
Not later. This practically doesn't make any sense to do. We are framework. Developers will add their indexes to code. What is the point of adding index command to their command line?

Reply:
I knew this will sping some good conversation.

Reply:
> You seem right

btw.. I'm always wrong :) question everything I say.

Reply:
yup users of CLI don't need to know indexes and don't want to add/remove indexes. Its developer's headache.

Reply:
we are not having any single index, we provide freedom to the developer to host their indexes and include them. And for this purpose hardcoding suits.

Reply:
Yes. to make it clear - multiple indexes as feature is good - but in SDK. This PR is all about end user Commands. 
There is separate issue to cover SDK #18